src/ViewerComponents/ResourceViewer.svelte

```diff
@@ -0,0 +1,415 @@
+<script>
+    import {onMount} from 'svelte';
+    import {token, lists, ontologies} from '../store';
+
+    export let resource, ontology, server, user, shortname, shortcode;
+    let properties = {}
+    let error = false;
+
+    onMount(async () => {
+        // For later
+        // await login();
+    })
+
+    /**
+     * Logs in to the provided server and saves the token into the store.
+     *
+     * @returns {Promise<void>}
+     */
+    async function login() {
+        error = false;
+
+        const res = await fetch(`https://${server}/v2/authentication`,
+            {
+                headers: new Headers({
+                    'Content-Type': 'application/json'
+                }),
+                method: 'POST',
+                body: JSON.stringify({'email': user['Email'], 'password': user['Pwd']})
+            })
+
+        // Checks if request succeeded
+        if (!res.ok) {
+            error = true;
+            console.error(res);
+            return;
+        }
+
+        const json = await res.json();
+        token.set(json.token);
+
+        await listRequest();
+    }
+
+    /**
+     * Requests all the list of the specific project and saves it into the store.
+     *
+     * @returns {Promise<void>}
+     */
+    async function listRequest() {
+        const res = await fetch(`https://${server}/admin/lists?${new URLSearchParams({projectIri: 'http://rdfh.ch/projects/' + shortcode})}`, {
+            headers: new Headers({
+                'Authorization': `Bearer ${$token}`
+            })
+        });
+
+        // Checks if request succeeded
+        if (!res.ok) {
+            error = true;
+            console.error(res);
+            return;
+        }
+
+        const json = await res.json();
+        lists.set(json.lists);
+
+        await ontologyRequest();
+    }
+
+    /**
+     * Requests the ontology of the specific project and saves it into the store.
+     *
+     * @returns {Promise<void>}
+     */
+    async function ontologyRequest() {
```

Comment:
Similar methods are already implemented in dsp-services

Reply:
Will happen later